components/clp-mcp-server/clp_mcp_server/server/constants.py

```diff
@@ -21,8 +21,10 @@
 "- Avoid using wildcards (`*`) unless absolutely necessary, as they can lead to large result"
 " sets.\n"
 "- Use logical operators (`AND`, `OR`, `NOT`) to combine one or more key-value searches.\n"
-"- Consider the time range of the logs you are searching. If the user specifies a time range,"
-" include it in the KQL query.\n"
+"- Consider the time range of the logs you are searching. Call"
+" `search_by_kql_with_timestamp_range` with a KQL query plus and explicit begin and end timestamps."
+" Timestamps must be in ISO 8601 format and end with `Z` to indicate UTC"
+" e.g. YYYY-MM-DDTHH:mm:ss.fffZ.\n"
```

Comment:
```suggestion
"- Consider specifying a time range to narrow down the search. Use"
" `search_by_kql_with_timestamp_range` with your KQL query and explicit start and end timestamps."
" Timestamps must follow the ISO 8601 UTC format (`YYYY-MM-DDTHH:mm:ss.fffZ`), where the trailing"
" `Z` indicates UTC.\n"
```
Do u think this is more clear?
Btw, ur version is not properly formatted as the line wrapping is a bit random.